src/main/java/uk/gov/ons/ssdc/supporttool/endpoint/JobEndpoint.java

```diff
@@ -168,6 +178,48 @@ public String getErrorDetailCsv(
     return csvContent;
   }
 
+  @PostMapping
+  public ResponseEntity<?> submitUploadJob(
+      @RequestParam(value = "fileId") String fileId,
+      @RequestParam(value = "fileName") String fileName,
+      @RequestParam(value = "collectionExerciseId") UUID collectionExerciseId,
+      @Value("#{request.getAttribute('userEmail')}") String userEmail) {
+
+    // Check that collex exists
+    Optional<CollectionExercise> collexOpt =
+        collectionExerciseRepository.findById(collectionExerciseId);
+    if (!collexOpt.isPresent()) {
+      throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Collection exercise not found");
+    }
+
+    // Check user is authorised to submit job for this survey
+    userIdentity.checkUserPermission(
+        userEmail, collexOpt.get().getSurvey(), UserGroupAuthorisedActivityType.LOAD_SAMPLE);
+
+    File file = new File(fileUploadStoragePath + fileId);
```

Comment:
this is a bit of an injection attack problem... might ignore it because it's just support tool but would be better to fix it now I think